gitmessage

```diff
@@ -1,11 +1,20 @@
+# 50-character subject line
 
+# 72-character wrapped description.
 
```

Comment:
I'm not sure we need this line. The 72-character length is enforced by other mechanisms. Having the line here could be confusing?